admin/templates/configuration/tmp/configure/mariadb.sh

```diff
@@ -2,14 +2,14 @@
 
```

Comment:
Keep the old name of this file for now? I think ideally we should be able to switch between mysql/mariadb quickly by changing one line in qserv.table file, in reality it could be more complicated but we should try to support both for now until we decide that we do not want to support mysql any more. "mysql" in my view is more generic term than mariadb, think of it as mariadb being one particular implementation of mysql. We still have all those MYSQL_* names everywhere, if we want to do renaming we should probably rename those too?